.changes/v2.15.0/438-features.md

```diff
@@ -0,0 +1,6 @@
+* Added support to set, get and delete metadata to AdminOrg with the methods
+  `AdminOrg.AddMetadataEntry`, `AdminOrg.AddMetadataEntryAsync`, `AdminOrg.GetMetadata`,
+  `AdminOrg.DeleteMetadataEntry` and `AdminOrg.DeleteMetadataEntryAsync`. [GH-438]
+* Added support to set, get and delete metadata to Disk with the methods
+  `Disk.AddMetadataEntry`, `Disk.AddMetadataEntryAsync`, `Disk.GetMetadata`,
+  `Disk.DeleteMetadataEntry` and `Disk.DeleteMetadataEntryAsync`. [GH-438]
```

Comment:
Nit: not sure we had multiline entries in changelog before, and whether the link parsing ("[GH-438]") will work correctly with make-changelog.sh - maybe you could double check?